cmake/Modules/FindCOIN.cmake

```diff
@@ -143,7 +165,6 @@ find_package_handle_standard_args(COIN DEFAULT_MSG
 
 # export the libraries
 if(COIN_FOUND)
-  set(COIN_INCLUDE_DIRS ${COIN_INCLUDE_DIR})
```

Comment:
why can this be removed?

Reply:
we have a target based approach now. Actually we could add it back. I thought it caused an error. But none of those vars are used anymore.